game/scripts/npc/abilities/ancient_apparition_ice_blast.txt

```diff
@@ -27,15 +27,15 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "40.0"
+    "AbilityCooldown"                                     "60.0" //OAA
```

Comment:
the rest of the values are already pretty rough for AA, but this cooldown is a super overnerf. I think you *either* change the cooldown like this *or* you change the values in the way you did (and maybe add 5-10s, not 20s)

Reply:
Please dont nerf AA in any way. He is fine right now. His spells should be strong

Reply:
He is absolutely not fine as he is lmao. It's general consensus he's the best hero in the game right now.